### PR TITLE
DOC: add issue and PR templates, zenodo.json, contributing

### PR DESCRIPTION
**Pull request by jklenzing**
Addresses #5 

Adds github templates for issues and pull requests.  Based on the versions at pysat, with a few additions for the examples.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Version [e.g. 22]
```
Comment (aburrell) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]:
```suggestion
 - Version [e.g. Python 3.7]
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+- [ ] Make sure you are merging into the ``develop`` (not ``master``) branch
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published in downstream modules
+- [ ] Add a note to ``CHANGELOG.md``, summarizing the changes
```
Comment (aburrell) [inline .github/pull_request_template.md:44]: Is there an Authors list or Zenodo authors json?  That could go on the checklist, for people to add themselves there.

Comment (jklenzing) [reply .github/pull_request_template.md:44]: There is not, but it would make sense to add that as part of this pull

Comment (jklenzing) [reply .github/pull_request_template.md:44]: Added the zenodo file based on the main pysat version, then updated based on the contributors here.

**Comment by jklenzing** [comment]
Since this almost completely addresses #5, I went ahead and added the rest of the docs here.

**Review by aburrell — APPROVED** [review]

**Pull request merged**